src/predict_3dpose.py

```diff
@@ -75,12 +75,19 @@
   'batch_normalization' if FLAGS.batch_norm else 'no_batch_normalization',
   'use_stacked_hourglass' if FLAGS.use_sh else 'not_stacked_hourglass',
   'predict_14' if FLAGS.predict_14 else 'predict_17')
-
+train_dir='train'
```

Comment:
I don't think we should overwrite `train_dir`. The directory structure is important for tensorboard to keep track of different parameter settings.